frida_tools/tracer.py

```diff
@@ -573,6 +603,21 @@ def stop(self):
             self._script = None
 
     def _create_trace_script(self):
+        # Extract all init session files into a single string,
+        # substitute newlines with '\u000a'
+        self._init_session_code = ''
+        for path in self._init_session_paths:
+            # Append contents of <path> to <self._init_session_code>
+            fo = open(path, "rt")
+            line = fo.readlines()
+            fo.close()
+
+            for i in range (len(line)):
+                line [i] = line [i].rstrip ()
```

Comment:
Extra spaces in function calls. (See style guide mentioned earlier.)

But how about using list comprehension here? E.g. `lines = [line.rstrip() for line in lines]`
(And renaming to `lines`, as it's more than one line.)

Reply:
Good use of comprehensions (although I've been using Python for a decade, I still think in C <g>).

I've made the line -> lines change.